ocs_ci/utility/lvmo_utils.py

```diff
@@ -56,3 +62,38 @@ def lvmo_health_check(tries=20, delay=30):
         delay=delay,
         backoff=1,
     )(lvmo_health_check_base)()
+
+
+def get_disks_by_path():
```

Comment:
We should be able to pass the node name to the func in case of multi-node sno cluster

Reply:
lets discuss it offline - im not sure how lvmcluster cr should look with multiple workers